euth/reports/emails.py

```diff
@@ -0,0 +1,72 @@
+from django.contrib.auth import get_user_model
+from django.contrib.sites.models import Site
+from django.core import urlresolvers
+from django.db.models import ForeignKey
+from django.template import Context
+
+from euth.contrib import emails
+
+
+def send_email_to_moderators(request, report):
+
+    obj = report.content_object
+
+    if obj:
+        name = obj._meta.verbose_name.title()
```

Comment:
Are you sure you want to use `title()` here? `verbose_name` should already have proper spelling.